dpnp/backend/kernels/dpnp_krnl_logic.cpp

```diff
@@ -51,17 +51,17 @@ DPCTLSyclEventRef dpnp_all_c(DPCTLSyclQueueRef q_ref,
     }
 
     sycl::queue q = *(reinterpret_cast<sycl::queue*>(q_ref));
-    sycl::event event;
 
     DPNPC_ptr_adapter<_DataType> input1_ptr(q_ref, array1_in, size);
     DPNPC_ptr_adapter<_ResultType> result1_ptr(q_ref, result1, 1, true, true);
```

Comment:
Could you please try to switch the first one `true` to `false` (parameter `target_no_sycl` = `false`)?
Then try to remove code with `DPNPC_ptr_adapter` at all.